custom_components/tahoma/number.py

```diff
@@ -80,13 +110,3 @@ async def async_set_value(self, value: float) -> None:
         await self.executor.async_execute_command(
             self.entity_description.command, value
         )
-
-    @property
-    def min_value(self) -> float:
-        """Return the minimum value."""
-        return self.entity_description.min_value or self._attr_min_value
-
-    @property
-    def max_value(self) -> float:
-        """Return the maximum value."""
-        return self.entity_description.max_value or self._attr_max_value
```

Comment:
Maybe we should try to keep this out of this PR. We have https://github.com/iMicknl/ha-tahoma/pull/667 ready for when 2022.2 drops.

Reply:
Will the step parameter be working  in the 2022.2  ?

Reply:
Yes :)